modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro

```diff
@@ -141,24 +163,24 @@ definition {
 			value1 = "\TAB");
 	}
 
-	@summary = "Default summary"
+	@summary = "It opens the Accessibility Menu."
 	macro openAccessibilityMenu() {
-		KeyPress(
-			locator1 = "//body",
-			value1 = "\TAB");
-
-		KeyPress(
-			key_link = "Skip to Main Content",
-			locator1 = "AccessibilityMenu#QUICK_LINK",
-			value1 = "\TAB");
+		AccessibilityMenu.goToAccessibilityMenuQuickLink();
 
 		KeyPress(
 			key_link = "Open Accessibility Menu",
 			locator1 = "AccessibilityMenu#QUICK_LINK",
 			value1 = "\ENTER");
 	}
 
-	@summary = "Default summary"
+	@summary = "It the user's profile menu."
+	macro OpenUserProfileMenu() {
+		Navigator.openURL();
```

Comment:
Is opening the URL necessary here? I think this is not needed and the added macro is not needed. Only UserBar.gotoDropdown can be called in the other macros.

Reply:
I just removed it  and got an error because of that.

![image](https://github.com/liferay-platform-experience/liferay-portal/assets/63858122/7d9e286f-ea7a-4d6c-946f-b39eaa3c9cd4)

Reply:
It seems safer to me to add `Navigator.openURL();` instead of closing the modal because in some parts the avatar for profile menu is not available. What do you think?

Reply:
Or we should actually add a Click to the close button? 🤔